htdocs/contact/class/contact.class.php

```diff
@@ -1283,18 +1295,6 @@ public function delete($notrigger = 0)
 			}
 		}
 
-		if (!$error) {
-			// Remove Roles
-			$sql = "DELETE FROM ".MAIN_DB_PREFIX."societe_contacts WHERE fk_socpeople = ".((int) $this->id);
```

Comment:
Why removing this ?
If we delete the contact, we must also delete it into this table, right ?